actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -41,11 +41,13 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      DOCUMENT = Nokogiri::HTML::Document.new
```

Comment:
I don't think this should be a constant; the creation of new nodes using `DOCUMENT.create_element` in `#render` will end up being a slow memory leak.

I think it's better to incur the cost of a `Document` object creation  in `#render` than to leak on `Node` object each call.